.pre-commit-config.yaml

```diff
@@ -55,16 +55,19 @@ repos:
       name: add-helm-version
       entry: ./scripts/add_helm_version.sh
       language: script
+      stages: [pre-push]
```

Comment:
Can we remove these stages again? Not sure why they were added

Reply:
Yes. I’ll remove the explicit ‘stages’ entries and restore the original defaults so this PR doesn’t alter hook stages.